sonic-xcvrd/xcvrd/xcvrd_utilities/port_mapping.py

```diff
@@ -119,13 +125,18 @@ def subscribe_port_update_event(namespaces):
             db = daemon_base.db_connect(list(d.keys())[0], namespace=namespace)
             asic_id = multi_asic.get_asic_index_from_namespace(namespace)
             port_tbl = swsscommon.SubscriberStateTable(db, list(d.values())[0])
+            port_tbl.db_name = list(d.keys())[0]
+            port_tbl.table_name = list(d.values())[0]
+            port_tbl.filter = d['FILTER'] if 'FILTER' in d else None
             asic_context[port_tbl] = asic_id
             sel.addSelectable(port_tbl)
+            logger.log_warning("subscribing to port_tbl {} - {} DB of namespace {} ".format(
+                                        port_tbl, list(d.values())[0]), namespace)
```

Comment:
Misaligned ')' causing script to bail out

Reply:
Agreed.

Reply:
now fixed